roles/validations/README.md

```diff
@@ -0,0 +1,14 @@
+# validations
+The QE Validations role provides a framework for performing state verification.
+This is useful for QE purposes and will be leveraged post-deployment to assert
+that required conditions have been satisfied. For example, if I update a certain
+value in my `OpenStackDataPlaneNodeSet`. I want to validate that the change has
+been applied as per my expectations on the relevant OpenStack Compute nodes.
+
+## Privilege escalation
```

Comment:
I guess this will depend on the validations. Putting a generic 
`Check the validations set you want to run for privilege escalations` may be something good in here.
Thoughts?